lib/shanty/plugins/rubocop_plugin.rb

```diff
@@ -6,6 +6,7 @@ module Plugins
     class RubocopPlugin < Plugin
       provides_projects_containing '**/.rubocop.yml'
       subscribe :test, :rubocop
+      description 'Excecutes Rubocop on Ruby projects where .rubocop.yml is present'
```

Comment:
Say `Discovers and runs` here too.

Also, just realised I don't know why we're restricting ourselves to just a `.rubocop.yml` here - we can do the same thing as Cucumber/RSpec where we check the dependencies to see if `rubocop` is in there. I'll raise an issue to fix that.